core/src/main/java/org/jobrunr/utils/mapper/jackson/modules/JobParameterDeserializer.java

```diff
@@ -25,9 +25,9 @@ public class JobParameterDeserializer extends StdDeserializer<JobParameter> {
 
     private final ObjectMapper objectMapper;
 
-    protected JobParameterDeserializer() {
+    protected JobParameterDeserializer(ObjectMapper objectMapper) {
         super(JobParameter.class);
-        this.objectMapper = new ObjectMapper();
+        this.objectMapper = objectMapper == null ? new ObjectMapper() : objectMapper;
```

Comment:
I don't see a way that `objectMapper` can be null so I think we can just assign it and the ternary function can be removed.